app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Auth;
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Models\TaskCategory;
+use Illuminate\Support\Facades\Validator;
+use App\Http\Requests\TaskCategoryStoreRequest;
+
+class TaskCategoriesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = User::where('id', Auth::user()->id)->with('taskCategories')->get();
+        $categories = $user[0]->taskCategories;
```

Comment:
User is already available in auth, no need to find it again. The code can be simply...

```php
$categories = auth()->user()->taskCategories;
```